config/testreport/js/xlt.js

```diff
@@ -570,6 +570,13 @@
             });
         })();
 
+        // reload
+        (function triggerPageReload() {
+            $('#clear-input').click(function(event) {
+                window.location.reload();
+            })
+        })();
```

Comment:
Instead of doing a complete page reload, I'd suggest to simply clear the input and trigger the filtering.

Reply:
Now, that the button makes use of a proper CSS class, the used selector need to be adapted here too.